src/couch_server.erl

```diff
@@ -549,9 +554,12 @@ delete_db_file(RootDir, FullFilePath, Options) ->
     end.
 
 rename_on_delete(Original) ->
+    file:rename(Original, renamed_filename(Original)).
+
+renamed_filename(Original) ->
```

Comment:
It's actually "deleted_filename", unless you make `deleted` suffix customizable.